etc/kayobe/kolla.yml

```diff
@@ -116,10 +116,19 @@ kolla_build_blocks:
   sahara_engine_footer: |
     # Bring in hack to the vanilla plugin
     RUN pip install -U --no-deps git+https://github.com/stackhpc/sahara.git@stackhpc/7.0.2.1
+  monasca_api_footer: |
+    # NOTE(brtknr): Install upstream patch currently under review to
+    # support time range on the server side
```

Comment:
Please include a link otherwise we'll have to find it at some future date to check if this has been merged.